beam_jobs/base_model_validator.py

```diff
@@ -0,0 +1,208 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+import re
+
+import apache_beam as beam
```

Comment:
Shouldn't library imports be located after Oppia imports? This also leads me to a thought that we should check all possible lint checks and enable them for the newly added folders from start.

Reply:
Yep we do have lint checks, I just asked for an early push to speed up feedback.